### DISCUSSION TITLE
The path is not displayed

### DISCUSSION THREAD

**Discussion by mr-shifty** [nvim-tree/nvim-tree.lua#2817] [Q&A] [answered]
Instead of the path, ??? is displayed

root_folder_label = true,
![Screenshot_20240703_180233.jpg](https://github.com/nvim-tree/nvim-tree.lua/assets/118600823/dbba847a-e733-4506-a275-9e2fdae8dca3)

**Comment by alex-courtis** [comment, marked as answer]
That needs to be a string or a function to show a label: `:help nvim-tree.renderer.root_folder_label`

**Reply by mr-shifty** [reply]
it was fixed. The help says that bool should also work😒

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/118600823/9022cc72-efc0-4f17-9975-22600c561def)
